hazelcast/src/main/java/com/hazelcast/json/JsonSerializable.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.hazelcast.internal.management;
+package com.hazelcast.json;
```

Comment:
Why are we moving `JsonSerializable` into the public API? AFAIK it's only used for serializing data sent to MC.

Reply:
It's already public API because `LocalMapStats` (and similar) inherit from it and are returned from methods like `IMap#getLocalMapStats`. Cleaning that up might be the subject of a follow-up PR.